src/tests/unit/components/question-editor/QuestionEditor.spec.jsx

```
@@ -0,0 +1,52 @@
+import { getByPlaceholderText, getByTestId, render } from '@testing-library/react'
```

ESLint

[error] Replace `·getByPlaceholderText,·getByTestId,·render·` with `⏎··getByPlaceholderText,⏎··getByTestId,⏎··render⏎`

(prettier/prettier)